components/category-age.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <annot-explanation :explanation="explanation"></annot-explanation>
+
+    <annot-columns
+      :columns="columns"
+      :active-category="activeCategoryName"
+      @remove:column="$emit('remove:column', $event)"
+    ></annot-columns>
+
+    <annot-age-values
+      :active-category="activeCategoryName"
+      :columns="columns"
+      :data-table="dataTable"
+      @update:dataTable="$emit('update:dataTable', $event)"
+    ></annot-age-values>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "annotate_age",
+  data() {
+    return {
+      explanation: 'This is an explanation for how to annotate age',
+      activeCategoryName: 'Age'
```

Comment:
I presume this is a store value? (Likewise across the other `category-*` components.)